src/widgets/keyboard/lv_keyboard.c

```diff
@@ -243,6 +400,23 @@ void lv_keyboard_set_map(lv_obj_t * obj, lv_keyboard_mode_t mode, const char * m
     lv_keyboard_update_map(obj);
 }
 
+#if LV_USE_KEYBOARD_PINYIN
+void lv_keyboard_set_font(lv_obj_t * obj, const lv_font_t * font)
```

Comment:
This may not be needed

Reply:
> This may not be needed

This function(`lv_keyboard_set_font()`) is to set the font of candidate Chinese characters. Some Chinese characters cannot be displayed in the default font (lv_font_simsun_16_cjk), Just like the picture I sent earlier, some Chinese characters are displayed as square boxes.

Reply:
I mean it can be set outside:
```c
lv_obj_t * kb = lv_keyboard_create(lv_scr_act());
lv_obj_set_style_text_font(kb, my_font, 0);
```
and there is no need to wrap this function with a layer.

Reply:
> I mean it can be set outside:
> 
> ```c
> lv_obj_t * kb = lv_keyboard_create(lv_scr_act());
> lv_obj_set_style_text_font(kb, my_font, 0);
> ```
> 
> and there is no need to wrap this function with a layer.

You are right, i will delete this function

Reply:
I have deleted this function(`lv_keyboard_set_font()`)